kalite/distributed/urls.py

```diff
@@ -101,6 +101,9 @@
     url(r'^store/', include(kalite.store.urls)),
     # the following pattern is a catch-all, so keep it last:
 
+    # Let user to create the super user from browser
+    url(r'^create_superuser_from_browser/$', 'create_superuser'),
```

Comment:
In fact, it might be beyond the scope of this PR (how urgent is it?) but this could be an opportunity to create more general RESTful API endpoints for creating & modifying users (or at least Django users if not also FacilityUsers).